assets/wire/components/filters/NavCreatedPicker.jsx

```diff
@@ -24,15 +24,17 @@ function NavCreatedPicker({setCreatedFilter, createdFilter, context}) {
     const activeShortcut = shortcuts.find((shortcut) => shortcut.value === createdFilter.from);
 
     return (
-        <NavGroup label={context === 'agenda' ? (gettext('Event Date')) : (gettext('Published'))}>
-            {shortcuts.map((shortcut) => (
-                <NavLink key={shortcut.value}
-                    label={shortcut.label}
-                    onClick={onClickFactory(shortcut.value)}
-                    isActive={shortcut === activeShortcut}
-                />
-            ))}
-            <div className="formGroup">
+        <NavGroup label={context === 'agenda' ? (gettext('Event date')) : (gettext('Published'))}>
```

Comment:
This should be `Event Date` and not `Event date` (notice the uppercase letter in date)